west.yml

```diff
@@ -198,7 +198,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 150b98fb2632d2660c8eedb5f992bcc72661fdc1
+      revision: pull/415/head
```

Comment:
You can delete this change.

Reply:
Not yet. The Zephyr side PR still needs to merge. When that occurs I will update his PR (and a couple of others)